10up/debug-bar-elasticpress
# Single button to copy all the info

### Description of the Change
This PR will replace the copy cURL buttons into a single copy button that will be used to copy the whole data instead of single cURL request. The position is also changed from bottom of the cURL request to top of all the cURL calls for better visibility and usage


Closes #59 

### How to test the Change
- Install Debug bar and Debug bar ElasticPress along with ElasticPress. 
- Go to the website and search for anything after activating ElasticPress.
- Enable query logging
- From the top right corner: Click on Debug->ElasticPress
- Notice that the multiple copy button will now be replaced by single copy button
- Click on it and it will copy all the cURL requests of the page


### Changelog Entry


> Changed - Copy link per query to a single copy button to copy all cURL requests instead of individual queries. 

### Credits

Props @MARQAS 

### Checklist:
